packages/contracts/contracts/L1/ZKVerifier.sol

```diff
@@ -715,14 +1190,21 @@ contract ZKVerifier {
 
     function verify(
         uint256[] calldata proof,
-        uint256[] calldata target_circuit_final_pair
+        uint256[] calldata target_circuit_final_pair,
+        bytes32 publicInputHash
```

Comment:
@chokobole 
As we discussed in #75, I modified this `verify` function to use `publicInputHash` as an argument.